src/Sdk/DTWebApi/WebApi/ActionsStepTelemetry.cs

```diff
@@ -15,6 +16,12 @@ public ActionsStepTelemetry()
             this.ErrorMessages = new List<string>();
         }
 
+        [DataMember(EmitDefaultValue = false)]
+        public string DisplayName { get; set; }
```

Comment:
the displayname might have EUII data since it might run through expression.

Reply:
why do we need to have the displayname as part of the telemetry?

Reply:
so, I guess... since we don't want it as part of telemetry, I need some other way... 🤔

Reply:
i would create a new class for your usage and not reuse the `ActionsStepTelemetry.cs`

Reply:
yeah makes sense will do

Reply:
addressed!